dev/AnimatedVisualPlayer/AnimatedVisualPlayer.idl

```diff
@@ -62,6 +88,9 @@ unsealed runtimeclass AnimatedVisualPlayer
     [MUX_DEFAULT_VALUE("true")]
     [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
     Boolean AutoPlay;
+    [MUX_DEFAULT_VALUE("winrt::AnimationsCacheModeEnum::Always")]
```

Comment:
You would need to version these APIs since you are adding to an existing type. See MUX_PUBLIC_V2 to see how it is done in other types. @kmahone as FYI

Reply:
Ok, done